cli/js/compiler.ts

```diff
@@ -32,8 +32,8 @@ function getAsset(name: string): string {
 // Constants used by `normalizeString` and `resolvePath`
 const CHAR_DOT = 46; /* . */
 const CHAR_FORWARD_SLASH = 47; /* / */
-const ASSETS = "$asset$";
-const OUT_DIR = "$deno$";
+const ASSETS = "asset://";
+const OUT_DIR = "deno://";
```

Comment:
Changed because TS was complaining during incremental compile that all paths must be either absolute or relative